erizo_controller/erizoClient/src/Room.js

```diff
@@ -682,14 +682,22 @@ const Room = (altIo, altConnectionHelpers, altConnectionManager, specInput) => {
   };
 
   // Returns callback(id, error)
-  that.startRecording = (stream, callback = () => {}) => {
+  that.startRecording = (stream, callbackOrOptions) => {
```

Comment:
Even if it's an API just to test things, I'd rather have 
```
that.startRecording = (stream, callback, advancedOptions=undefined) => {
```
We can decide later at what point and in what format we can expose that options in the API and update the docs accordingly.

Reply:
@lodoyun yes, totally, I've even forgot you can add arbitrary params to a js fn, I'll update as soon as I can.